From: Andrew Morton <akpm@linux-foundation.org>
Subject: delay-replace-kernelh-with-the-necessary-inclusions-fix

drivers/media/dvb-frontends/cxd2880/cxd2880_common.h needs bits.h for GENMASK()

Reported-by: kernel test robot <lkp@intel.com>
Cc: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 drivers/media/dvb-frontends/cxd2880/cxd2880_common.h |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/media/dvb-frontends/cxd2880/cxd2880_common.h~delay-replace-kernelh-with-the-necessary-inclusions-fix
+++ a/drivers/media/dvb-frontends/cxd2880/cxd2880_common.h
@@ -12,6 +12,7 @@
 #include <linux/types.h>
 #include <linux/errno.h>
 #include <linux/delay.h>
+#include <linux/bits.h>
 #include <linux/string.h>
 
 int cxd2880_convert2s_complement(u32 value, u32 bitlen);
_
